common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -845,3 +845,202 @@ def test_generate_certificate_exception(self):
                              ' below to send the certificate.',
             self.certificates_section.message.text
         )
+
+
+@attr('shard_1')
+class CertificateInvalidationTest(BaseInstructorDashboardTest):
+    """
+    Tests for Certificates functionality on instructor dashboard.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(CertificateInvalidationTest, cls).setUpClass()
+
+        # Create course fixture once each test run
+        CourseFixture(
+            org='test_org',
+            number='335535897951379478207964576572017930000',
```

Comment:
course number is from [certificates_web_view fixture](https://github.com/edx/edx-platform/blob/master/common/test/db_fixtures/certificates_web_view.json#L69)